src/api/express.ts

```diff
@@ -1,12 +1,49 @@
-import express from 'express';
-import healthRouter from './routes/health-routes';
-const app = express();
+import express, { Application, Response as ExResponse, Request as ExRequest, NextFunction } from 'express';
+import morgan from 'morgan';
+import swaggerUi from 'swagger-ui-express';
+import { RegisterRoutes } from './routes/routes';
+import { ValidateError } from "tsoa";
+
+const app: Application = express();
 
 //Express middleware
+app.use(morgan("tiny"));
+app.use(express.json());
+app.use(express.static("public"));
 
 //Routing information
+app.use(
+    "/docs",

```

Comment:
If possibly, I'd like our api endpoints to originate from /api as the base route.

The convention that our projects have been using for serving swagger has been /api/api-docs, if possible I'd like to stick to this convention as well.

Reply:
Updated in last commit,